cornell-dti/course-plan
# Run Courses Script for FA23

### Summary 

This PR runs the courses script for FA23, fixing a few discrepancies users have noticed. Specifically, the course data collection gives us up to date information for every course, but we still source available courses themselves from the large courses JSON.


### Test Plan 

- Select a couple courses and ensure that data is up to date for Fall 2023
- Check that GERST 1126 exists (Hotjar feedback)
